README.md

```diff
@@ -1,5 +1,9 @@
 # Swift OpenAPI Generator Runtime
 
+[![](https://img.shields.io/badge/docc-read_documentation-blue)](https://swiftpackageindex.com/apple/swift-openapi-runtime/documentation)
```

Comment:
Would you want to add the SSWG badge too perhaps, or have you already thought about it?
With an `img` tag it would look like so:
```
<img src="https://user-images.githubusercontent.com/54685446/201329617-9fd91ab0-35c2-42c2-8963-47b68c6a490a.png" alt="SSWG sandbox">
```

Reply:
I see the generator repository does have it, which makes me think not including the badge here was intentional and was to just keep the badge to the main repo.